soc/riscv/riscv-privilege/nuclei/common/nuclei_csr.h

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright (c) 2019 Nuclei Limited. All rights reserved.
+ * Copyright (c) 2021 TOKITA Hiroshi
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __NUCLEI_CSR_H__
+#define __NUCLEI_CSR_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+#define UCODE_OV            (0x1)
+
+#define WFE_WFE                     (0x1)
+#define TXEVT_TXEVT                 (0x1)
+#define SLEEPVALUE_SLEEPVALUE       (0x1)
+
+#define MCOUNTINHIBIT_IR            (1 << 2)
+#define MCOUNTINHIBIT_CY            (1 << 0)
```

Comment:
nit: better use `BIT()` macro
remember to add `U` suffix to all unsigned values.

Reply:
Use BIT() macro and add U suffix to numeric literals.